docs/ota-client-guide/modules/ROOT/pages/virtual-secondaries.adoc

```diff
@@ -0,0 +1,45 @@
+= Use of Virtual Secondaries
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+
+Virtual secondaries can be used as a way to update a configuration file on the primary (where the primary is a standard OSTree image running an aktualizr). You can do this in two parts:
+
+* First you need a virtual secondary configured that can be used to update a configuration file.
+* Afterwards you can integrate it into your Yocto build.
+
+
+
+== Creating a Virtual Secondary
+
+. You can create a virtual secondary to run locally by following the steps on how to xref:simulate-device-basic.adoc[simulate a device without building a disk image].
++
+You can use the provided `.deb` releases mentioned and the `virtualsec.json` file, which you can edit to suit your needs: such as removing one of the two entries and setting the paths and names to whatever you prefer.
+. Afterwards, you will be able to download the arbitrary files to the location specified in the `firmware_path` variable.
+. Upload the files for the virtual secondary to the https://connect.ota.here.com[OTA Connect Portal].
+. Now go to the *Software versions* tab and click *Add Software*. You should make sure the *ECU Types* field matches the value specified in the `ecu_hardware_id` variable of the `virtualsec.json` file.

```

Comment:
These two steps ("Upload the files..." and "Now go...") look to me like the same thing. The first one is just the high-level description and the second one has the details. They can probably be combined on one line with just minor changes to the transition words.

Reply:
@patrickvacek  good point. I will make the change.